### PR TITLE
create endpoint for token multiplier

### LINKED ISSUE #77: Create endpoint for fetching mulitpier per token
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Set up the project, read [README.md](https://github.com/djeck1432/spotnet/blob/main/README.md#development-environment-setup) how to do that.
3. In [position.py](https://github.com/djeck1432/spotnet/blob/main/web_app/api/position.py) create new endpoint: `get_mulitpiers` which will return data like `{"ETH": <multilpier>}`
4. Create `dataclass` which is called `TokenMultipliers` in [constants.py](https://github.com/djeck1432/spotnet/blob/main/web_app/contract_tools/constants.py) with next values:
```
 ETH = 5
 STRK = 2.5
 USDC = 5
```
5. Add serializer response model for new created endpoint, `TokenMulitpliersResponse` in [posotion.py](https://github.com/djeck1432/spotnet/blob/main/web_app/api/serializers/position.py)

### PR DESCRIPTION
**Pull request by od-hunter**
Resolves #77 
@djeck1432 Kindly review the endpoint created for token multiplier.

### CODE REVIEW

File: web_app/api/position.py [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
+            }
+        }
+    
+
+@router.get("/api/get-multipliers", tags=["Position Operations"], response_model=TokenMultiplierResponse, summary="Get token multipliers", response_description="Returns token multipliers")
+async def get_multipliers() -> TokenMultiplierResponse:
+    """
+    This Endpoint retrieves the multipliers for tokens like ETH, STRK, and USDC.
+    """
+
```
Comment (djeck1432) [inline web_app/api/position.py:36]: remove blank lines

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Looks really good, great jobs, small issues and ready to merge

File: web_app/api/position.py [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
 from web_app.api.serializers.position import PositionFormData
-from web_app.contract_tools.constants import TokenParams
+from web_app.contract_tools.constants import TokenParams, TokenMultipliers
 from web_app.contract_tools.mixins.deposit import DepositMixin
 from web_app.db.crud import PositionDBConnector
 
 router = APIRouter()  # Initialize the router
 position_db_connector = PositionDBConnector()  # Initialize the PositionDBConnector
 
+class TokenMultiplierResponse(BaseModel):
```
Comment (djeck1432) [inline web_app/api/position.py:16]: Add docstrings

File: web_app/api/position.py [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+    multipliers: dict[str, float]
+
+    class Config:
```
Comment (djeck1432) [inline web_app/api/position.py:19]: add docstring

File: web_app/contract_tools/constants.py [hunk 4]
```diff
@@ -23,6 +23,17 @@ class TokenConfig:
     name: str
 
 
+@dataclass(frozen=True)
+class TokenMultipliers:
+    """
+    Class to hold the predefined multipliers for supported tokens/
+    """
+
```
Comment (djeck1432) [inline web_app/contract_tools/constants.py:31]: remove blank line

**Comment by od-hunter** [comment]
@djeck1432 , please review

**Comment by djeck1432** [comment]
@od-hunter Please, fix pylint issues

**Comment by od-hunter** [comment]
@djeck1432  please review

**Review by djeck1432 — APPROVED** [review]
Looks good to me, Approved

**Pull request merged**